README.md

```diff
@@ -52,34 +52,36 @@ Number             | Title                                                    |
 [42](bsip-0042.md) | Adjust price feed to influence trading price of SmartCoins                  | Abit More | Protocol | Rejected
 [43](bsip-0043.md) | Market Fee Sharing                                                          | OpenLedgerApp | Protocol | Installed
 [44](bsip-0044.md) | Hashed Time-Locked Contract                                                 | Ryan R. Fox | Protocol | Installed
-[45](bsip-0045.md) | Add bitAsset Backing Collateral flag/permission                             | Customminer | Protocol | Draft
+[45](bsip-0045.md) | Add bitAsset Backing Collateral flag/permission                             | Customminer | Protocol | Voting
 [46](https://github.com/bitshares/bsips/pull/111) | Escrow Concepts                              | Taconator | Informational | Accepted
-[47](bsip-0047.md) | Vote Proxies for Different Referendum Categories and explicit voting operation                 | Fabian Schuh | Protocol | Draft
-[48](https://github.com/bitshares/bsips/pull/115) | Add Flag to Asset to Prevent Manipulating Max Supply | Fabian Schuh | Protocol | Draft
+[47](bsip-0047.md) | Vote Proxies for Different Referendum Categories and explicit voting operation                 | Fabian Schuh | Protocol | Voting
+[48](https://github.com/bitshares/bsips/pull/115) | Add Flag to Asset to Prevent Manipulating Max Supply | Fabian Schuh | Protocol | Voting
```

Comment:
Already voting? Please merge the draft into master. If already done, please update the link.